### DISCUSSION TITLE
下面的代码在提交OJ后出现了RE问题，但是在本地运行时我的测试样例与答案相符

### DISCUSSION THREAD

**Discussion by f-x-here** [datawhalechina/learn-python-the-smart-way-v2#24] [提问答疑] [answered]
<img width="711" alt="image" src="https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/143939882/69e277a9-21e9-4b2f-a665-140f364e9542">
预期出现结果是AC qwq

**Comment by anine09** [comment]
请仔细阅读提问规则，我无法理解你的提问

骆秀韬
聪明办法学 Python 教学团队
________________________________
From: f-x-here ***@***.***>
Sent: Monday, October 16, 2023 2:15:02 PM
To: datawhalechina/learn-python-the-smart-way-v2 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [datawhalechina/learn-python-the-smart-way-v2] 下面的代码在提交OJ后出现了RE问题，但是在本地运行时我的测试样例与答案相符 (Discussion #24)


[image]<https://user-images.githubusercontent.com/143939882/275407827-69e277a9-21e9-4b2f-a665-140f364e9542.png> 预期出现结果是AC qwq

―

**Reply by f-x-here** [reply]
运行环境：win11 python 3.1
报错信息：在本地运行时没有报错且对提供的测试样例返回了正确答案，但是在HydroOJ提交测评时出现了RE3
我希望看到的结果：AC
过程：在初次RE后，我认为有可能是a,b输入后为字符类型导致的RE，但是根据题中所给注释，应该没有问题，而加上了强制类型转换后，提交得到的结果依然是RE，编译器对于语法没有报错，且逻辑（在我看来）没有发现明显问题。
目前的思考：后台的测试数据给出的输入数据可能不是用“，”分隔？

1 reply
@f-x-here
f-x-here
now
Author
同时，在HydroOJ进行自测评时，得到也也是Accepted 33.403145ms 3396KiB，答案没有出现问题

**Comment by anine09** [comment, marked as answer]
你的猜测是对的，我们已经修复这个问题，请重新提交，感谢你报告这个bug。

骆秀韬
聪明办法学 Python 教学团队
________________________________
